sdk/cognitiveservices/azure-cognitiveservices-inkrecognizer/README.md

```diff
@@ -0,0 +1,145 @@
+# Azure Ink Recognizer client library for Python
+
+Azure Ink Recognizer SDK is an SDK for developers to work with Azure Ink Recognizer Service. The service recognize a collection of ink strokes and return a tree hierarchy of the recognized units, such as lines, words, shapes, as well as the handwriting recognition result of the words.
+
+Features:
+
+* Connect to Azure Ink Recognizer Service
+* Convert collections of ink strokes into HTTP requests
+* Parse HTTP response into ink recognition units
+
+[Source code][source_code] | [Package (PyPi)][pypi] | [API reference documentation][ref_inkrecognizer_sdk] | [Product documentation][ink_recognizer_docs] | [Samples][samples]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Cosmos DB client library for Python with [pip][pip]:
+
+```Bash
+pip install azure-cognitiveservices-inkrecognizer
+```
+
+**Prerequisites**: You must have an [Azure subscription][azure_sub]. You must have [Python 2.7][python] or [Python 3.5.3+][python] to use this package. Asynchronous features supports Python 3.5.3+ only.
+
+#### Get URL
+
+TODO.
+
+#### Get credentials
+
+TODO.
+
+## Key concepts
+
+TODO.
+
+## Examples
+
+The [Samples][samples] provide several code snippets covering some of the most common Ink Recognizer SDK tasks, including:
+
+* Implement InkPoint and InkStroke classes
+* Convert stroke unit from pixel to mm
+* Set language recognition locale
+* Indexing a key word from recognition results
+* Set application kind if user know expected type of ink content
+
+### Implement ink stroke
+
+If you don't have any pre-defined ink point or ink stroke classes, you can either follow the [Ink Stroke Interfaces][ink_stroke_file] to build your stroke, or build your own class that has all required fields. If you already defined ink strokes yourself, you should feed attributes in your class according to the interfaces.
+
+```Python
+from azure.cognitiveservices.inkrecognizer import InkStrokeKind
+
+InkPoint = namedtuple("InkPoint", "x y")
+
+class InkStroke():
+    def __init__(self,
+                 ink_stroke_id,
+                 ink_points,
+                 stroke_kind=InkStrokeKind.UNKNOWN,
+                 stroke_language=""):
+        self.id = ink_stroke_id
+        self.points = ink_points
+        self.kind = stroke_kind
+        self.language = stroke_language
+```
+
+You can then create a list (or any iterable object) of ink strokes for recognition.
+
+### Create a client
+
+Once you got the url for ink recognizer service and an Azure credential instance, you can create an InkRecognizerClient
+
+```Python
+from azure.cognitiveservices.inkrecognizer import InkRecognizerClient
+client = InkRecognizerClient(url, credential)
```

Comment:
Since as of today you support key only, I would an example:

```python
from azure.cognitiveservices.inkrecognizer import InkRecognizerClient
client = InkRecognizerClient(url, api_key)  # api_key is your key as string
```